internal/testing/releasehelper.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Helper tool for creating new releases of the Go CDK.
+// Run from the repository root:
+// $ go run internal/testing/releasehelper.go <flags>
+package main
```

Comment:
Add a simple test?

Reply:
I'm not sure it's worth it. To really test it I'll need to recreate a small vistual file system since it invokes the go tool to actually modify files, etc. This test is going to be much more complicated than the tool itself. WDYT?

Reply:
OK for now; may need to revisit as it gets more complicated.